src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/SqlClientCustomTokenCredential.cs

```diff
@@ -31,11 +34,11 @@ private async Task<AccessToken> AcquireTokenAsync()
         {
             // Added to reduce HttpClient calls.
             // For multi-user support, a better design can be implemented as needed.
-            if (_akvUrl != DataTestUtility.AKVUrl)
+            if (_akvUrl != AKVUrl)
```

Comment:
as far as I can tell this variable isn't being used to do anything other than indicate that the token has been acquired. If that's true, I'd recommend either making it a bool or checking `_authority`/`_resource` for `null` as the check. 

But I'm also wondering, can we store the access token itself? Or does that need to be re-acquired each time the call to this method is made? If we can store the access token, we could use `_accessToken` as the value to check here (plus we could get rid of `_authority` and `_resource`.

Reply:
I agree. AKVUrl was used all over the place for a static AKV credential, this was the last referenced location. It's now gone!

I'm not sure about the impact that caching the access token could have on test coverage - only the first test to run would ever be able to exercise the case where the AKV access token can't be retrieved. I think it'd be better to explicitly add a TokenCredential derivative which throws and run some of the AKV tests based on that, then revisit any caching.